test/functional/feature_llmq_is_cl_conflicts.py

```diff
@@ -71,16 +80,16 @@ def run_test(self):
         self.test_chainlock_overrides_islock(False)
         self.test_chainlock_overrides_islock(True, False)
         self.test_chainlock_overrides_islock(True, True)
-        self.test_chainlock_overrides_islock_overrides_nonchainlock(False)
-        self.activate_dip0024()
-        self.log.info("Activated DIP0024 at height:" + str(self.nodes[0].getblockcount()))
-        self.test_chainlock_overrides_islock_overrides_nonchainlock(False)
+        self.test_chainlock_overrides_islock_overrides_nonchainlock()
+        #self.activate_dip0024()
+        #self.log.info("Activated DIP0024 at height:" + str(self.nodes[0].getblockcount()))
+        self.test_chainlock_overrides_islock_overrides_nonchainlock()
         # At this point, we need to move forward 3 cycles (3 x 24 blocks) so the first 3 quarters can be created (without DKG sessions)
         self.move_to_next_cycle()
         self.move_to_next_cycle()
         self.move_to_next_cycle()
         self.mine_cycle_quorum()
-        self.test_chainlock_overrides_islock_overrides_nonchainlock(True)
+        self.test_chainlock_overrides_islock_overrides_nonchainlock()
```

Comment:
I think this part (lines 84-92) is no longer needed cause we already test the only available option at line 83